Dev/VideoChatClient/AutoInitWinSock/AutoInitWinSock.cpp

```diff
@@ -0,0 +1,25 @@
+#include "AutoInitWinSock.hpp"
+
+AutoInitWinSock::AutoInitWinSock()
+{
+	WSADATA wsaData{};
+	if (0 == WSAStartup(MAKEWORD(2, 2), &wsaData))
```

Comment:
Have you read the Documentation of WSAStartup? This could have caused some fistunim 🐱‍🏍

Reply:
DO you mean I need to call WSACleanup in the second if?